src/controller/datacontroller.js

```diff
@@ -180,7 +183,10 @@ export default class DataController {
 
 	/**
 	 * Sets input data parsed by the {@link #processor data processor} and
-	 * converted by the {@link #viewToModel view to model converters}.
+	 * converted by the {@link #viewToModel view to model converters}. If markers where converted
+	 * from view to model as temporary {@link module:engine/model/element/~Element model elements} then those elements
+	 * will be removed from parsed {@link module:engine/model/element/~DocumentFragment} and added to the
+	 * {@link module:engine/model/document~Document#markers markers collection}.
```

Comment:
Especially since now `parse` does not return any useful data, and the whole processing is done outside this function.